docs/search/getting-started.md

```diff
@@ -0,0 +1,68 @@
+---
+sidebar_position: 20
+title: Getting started with Search
+---
+
+In this getting started guide, you will add search functionality to the Address Book app that you build in the [Macrometa Quickstart](https://macrometa.com/docs/quickstart).
+
+## Prerequisites
+
+- A [Macrometa account](https://auth.paas.macrometa.io/).
```

Comment:
@elof this is the old URL to login correct? It should be auth-play.macrometa.io I believe

Reply:
@dallas-macrometa @diana-macrometa, yes this is the old url

Reply:
Is it supposed to be?